hpat/tests/test_series.py

```diff
@@ -3627,6 +3627,75 @@ def test_impl(s, axis):
             msg = 'Method cumsum(). Unsupported parameters. Given axis: int'
             self.assertIn(msg, str(raises.exception))
 
+    def test_series_getitem(self):
+        def test_impl(S, key):
+            return S[key]
+
+        jit_impl = hpat.jit(test_impl)
+
+        keys = [2, 2, '2']
+        indices = [[2, 3, 5], ['2', '3', '5'], ['2', '3', '5']]
+        for key, index in zip(keys, indices):
+            S = pd.Series([11, 22, 33], index)
+            self.assertEqual(jit_impl(S, key), test_impl(S, key))
+
+    def test_series_getitem_slice(self):
+        def test_impl(S, start, end):
+            return S[start:end]
+
+        jit_impl = hpat.jit(test_impl)
+
+        starts = [0, 0, '2']
+        ends = [2, 2, '5']
+        indices = [[2, 3, 5], ['2', '3', '5'], ['2', '3', '5']]
+        for start, end, index in zip(starts, ends, indices):
+            S = pd.Series([11, 22, 33], index)
+            ref_result = test_impl(S, start, end)
+            print(type(ref_result))
```

Comment:
I forgot to remove the debug print. @1e-to could you please remove it with the next patch?